static/app/components/organizations/timeRangeSelector/pageTimeRangeSelector.tsx

```diff
@@ -0,0 +1,83 @@
+import {ComponentProps, useState} from 'react';
+import styled from '@emotion/styled';
+
+import TimeRangeSelector from 'app/components/organizations/timeRangeSelector';
+import {Panel} from 'app/components/panels';
+import {DEFAULT_RELATIVE_PERIODS, DEFAULT_STATS_PERIOD} from 'app/constants';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+
+type Props = Omit<
+  ComponentProps<typeof TimeRangeSelector>,
+  'defaultPeriod' | 'onChange'
+> & {
+  className?: string;
+  defaultPeriod?: ComponentProps<typeof TimeRangeSelector>['defaultPeriod'];
+  onChange?: ComponentProps<typeof TimeRangeSelector>['onChange'];
+};
+
+function PageTimeRangeSelector({className, ...props}: Props) {
+  const [isCalendarOpen, setIsCalendarOpen] = useState(false);
+
+  return (
+    <DropdownDate className={className} isCalendarOpen={isCalendarOpen}>
+      <TimeRangeSelector
+        label={<DropdownLabel>{t('Date Range:')}</DropdownLabel>}
+        onToggleSelector={isOpen => setIsCalendarOpen(isOpen)}
+        defaultPeriod={DEFAULT_STATS_PERIOD}
+        onChange={() => {}}
```

Comment:
should we update the `TimeRangeSelector` component, making this prop optional?

Reply:
done!